src/services/models/Schema.ts

```diff
@@ -244,7 +244,7 @@ function buildFields(
     sortByRequired(fields, schema.required);
   }
 
-  if (typeof additionalProps === 'object') {
+  if (typeof additionalProps === 'object' || typeof additionalProps === 'boolean') {
```

Comment:
if `additionalProperties === false` you should not push "additional properties" field, change this to checking if it is implicit `true`:

```js
if (typeof additionalProps === 'object' || additionalProps === true) {
```

Reply:
Implemented.